src/schemas/OrganizationApiTokenCreateRequest.yaml

```diff
@@ -1,11 +1,15 @@
 type: object
 required:
   - name
-  - scope
+  - roleId
 properties:
   name:
     type: string
   description:
     type: string
   scope:
     $ref: './enums/OrganizationApiTokenScope.yaml'
+  roleId:
+    type: string
+    format: uuid
+    nullable: true
```

Comment:
description: the roleId provided by the "List organization custom roles" endpoint.

Reply:
done